.murdock

```diff
@@ -457,24 +457,33 @@ compile() {
                         make -C${appdir} all test-input-hash -j${JOBS:-4}
         RES=$?
         if [ $RES -eq 0 ]; then
-            kconfig_test_hash=$(test_hash_calc "${BINDIR}")
+            kconfig_hashes="$(cat ${BINDIR}/test-input-hash.sha1)"
         else
-            kconfig_test_hash=0
+            kconfig_hashes="kconfig-build-failed"
             echo "An error occurred while compiling using Kconfig";
         fi
     fi
 
     # compile without Kconfig
     CCACHE_BASEDIR="$(pwd)" BOARD=$board TOOLCHAIN=$toolchain RIOT_CI_BUILD=1 \
-        make -C${appdir} clean all test-input-hash -j${JOBS:-4}
+        make -C${appdir} clean
```

Comment:
ok, now the clean gets all the env variables, but the actual compile doesn't.

I've split out this fix in https://github.com/RIOT-OS/RIOT/pull/18254.